run.py

```diff
@@ -28,7 +37,8 @@
         print("freezing")
         freezer.freeze()
         print("frozen")
-        freezer.serve(host=host, port=port, threaded=True)
+        if not no_run:
```

Comment:
If you  change line 18 to `run = !args.norun` then this could change to `if  run:`, which I think it much easier to understand. But this is alright and I'm not determined to change it.